tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
```

Comment:
question: why not add `set -xe` ?

Reply:
This is not automated and needs to be run manually, so I do not see a need for `x` at the moment but I will add the `e` option.